### PR TITLE
Added GitHub Actions

### PR DESCRIPTION
**Pull request by daminijain23**
Added - release.yml and tox-test.yml
Removed - .travis.yml

https://projects.engineering.redhat.com/browse/RHELDST-7276

### CODE REVIEW

**Comment by rbikar** [comment]
@daminijain23 
Looks good to me, but can you please check whether it's possible also to trigger JenkinsCI by GH Actions? Currently it's done by manually adding comment to PR like "run tests" or similar.

File: .github/workflows/tox-test.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      - name: Run Tox
+        run: tox -e static
+  coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install RPM
+        run: |
+          sudo apt-get install -y rpm
+          sudo apt-get install -y libkrb5-dev
```
Comment (rbikar) [inline .github/workflows/tox-test.yml:77]: Is installation of libkrb5-dev required?

**Comment by rbikar** [comment]
@daminijain23 
I  added one comment about installation of some specific package, can you have a look? Otherwise this looks OK, and I would rather merge it, so we can test it properly and later add support for JENKINS CI trigger, if possible.

**Comment by rbikar** [comment]
@daminijain23 
I just wonder, why travis.yml is back. This file shouldn't be needed now. Or am I missing something?

**Comment by daminijain23** [comment]
@rbikar 
krb5 package have been removed from coverage flow as it wasn't required. 
.travis.yml was added back as backup in case github actions errors out and until all issues get resolved

**Comment by rbikar** [comment]
O

> @rbikar
> krb5 package have been removed from coverage flow as it wasn't required.
> .travis.yml was added back as backup in case github actions errors out and until all issues get resolved

OK, sounds good, I'll merge this now and let's see how it works :)

**Review by rbikar — APPROVED** [review]

**Pull request merged**